src/redux/api/rundownApi.ts

```diff
@@ -42,11 +42,40 @@ export const rundownApi = baseApi.injectEndpoints({
       }),
       invalidatesTags: (result) => [{ type: 'Rundown', id: result?.id }],
     }),
+    deleteRundown: builder.mutation<void, { id: string }>({
+      query: ({ id }) => ({
+        url: `/rundowns/${id}`,
+        method: 'DELETE',
+      }),
+      invalidatesTags: ['Rundown'],
```

Comment:
For caching you could invalidate, not the whole rundown but just the rundown for that ID. This way the system doesn't have to refetch the whole thing and just fetch for 1 detail rundown. 

What I meant is that something like this:

```
invalidatesTags:(_, __, arg) => [
        { type: 'Rundown', id: arg.id },
        'Timeline',
      ],
```